### ISSUE TITLE
inject problem stuck

### ISSUE THREAD

**Issue by zxcvqwerasdf** [Dewera/Lunar#43]
Target: steam.exe (x86)
dll: https://github.com/altoid29/VAC3_Dumper (x86)
Xenos inject it's with no problem, but not Lunar.

![image](https://user-images.githubusercontent.com/67195196/234674362-490ef127-64de-475b-b015-cb4b96e5c517.png)
Stuck on 183 line in ProcessContext.cs
Here is callstack:
![image](https://user-images.githubusercontent.com/67195196/234674587-047572f3-de28-4309-bdc0-400e7f15527a.png)
I waited 5 minutes, but nothing happening, it's just stuck in InitialiseTlsData on
```
 if (Kernel32.WaitForSingleObject(threadHandle, int.MaxValue) == -1)
 ```

**Comment by Dewera** [comment]
Can't replicate locally but sounds like a resource lock issue (Microsoft sometimes changes what is and isn't locked by internal locks.) To verify could you comment out line 430 (`using var pebLock = new PebLock(_processContext);`) and try again?

**Comment by zxcvqwerasdf** [comment]
Yep, commented ```using var pebLock = new PebLock(_processContext)``` makes it works
